crates/astria-conductor/src/executor/mod.rs

```diff
@@ -467,7 +467,7 @@ impl ExecutableBlock {
         let timestamp = convert_tendermint_to_prost_timestamp(block.header().time);
         let transactions = block
             .into_rollup_transactions()
-            .remove(&id)
+            .swap_remove(&id)
```

Comment:
Note: dependency bump required this update `remove` was deprecated and is equivalent to `swap_remove`